pycompwa/expertsystem/ui/default_settings.py

```diff
@@ -22,8 +23,10 @@
 default_particle_list_search_paths = [
     "../../../../Physics",
     "../../../..",
-    "../../"
+    "../../",
 ]
+default_particle_list_search_paths += [sys.prefix + '/share']
+default_particle_list_search_paths += sys.path
```

Comment:
I thought the particle_list.xml is in the ComPWA/Physics directory. So in dev mode it wont be found or?

Reply:
The first one
```python
default_particle_list_search_paths += [sys.prefix + '/share']
```
is in case you use `python setup.py install`. The second
```python
default_particle_list_search_paths += sys.path
```
is in case you run `pip install pycompwa`.

Yep.. it's a mess

Reply:
My bad, I misread this...